Client/States/GameplayState.cpp

```diff
@@ -725,3 +730,22 @@ float GameplayState::CalculateCompletion(Vector3 playerCurPos){
     auto progress = playerCurPos - startPos;
     return progress.Length()/levelLen;
 }
+
+void GameplayState::RenderFlairObjects(){
+    Vector3 lavaPos = deathPos ;
+
+    AddLava(lavaPos + Vector3(0, 5, 0));
+    AddEndPortal(endPos);
+
+}
+
+void GameplayState::AddLava(Vector3 position){
+    auto LavaQuad = new GameObject();
+    LavaQuad->GetTransform().SetOrientation(Quaternion(Matrix4::Rotation(90, {1,0,0})));
+    replicated->AddTestObjectToLevel(LavaQuad, *world, {1000,1000,1000}, position, false);
+    LavaQuad->SetRenderObject(new RenderObject(&LavaQuad->GetTransform(), resources->GetMesh("QuadSequel.msh"), resources->GetTexture("VorDef.png"), resources->GetShader("lava")));
```

Comment:
We should change this back to normal quad lmao

Reply:
deleted yes yes killed deleted